### DISCUSSION TITLE
Overlapping commits relating to a pull request

### DISCUSSION THREAD

**Discussion by grahamperrin** [community/community#35508] [Pull Requests]
With https://github.com/freebsd/freebsd-quarterly/pull/517/files (someone else's pull request) as it was on 7th October, I thought it might be convenient to suggest changes to the file in the form of my own PR. 

Coincidentally <https://github.com/freebsd/freebsd-quarterly/pull/517/commits> an 8th October commit landed whilst I was editing in my fork. So: 

* https://github.com/grahamperrin/freebsd-quarterly/commit/7a3b85eaf5348013b6dea940450e23851feb555c

– line 5, for example, I need to get in sync with <https://github.com/freebsd/freebsd-quarterly/pull/517/commits/53d64634e2a96061e6ad74adec56a34f716b0cbd>, which preceded my commit by a few minutes. 

Can I use a feature of GitHub to untangle the overlap? 

Thanks

**Comment by jfullstackdev** [comment]
what do you mean by that ? conflict in merge ? always sync the repo then made the changes in the fork, that way there is no overlapping commits

**Reply by grahamperrin** [reply]
> … conflict in merge ? …

No, I didn't proceed to a (draft) PR, I stopped when I realised (after my commit to my `patch-4` branch): 

![image](https://user-images.githubusercontent.com/192271/194713078-7fd3daa4-1825-4018-a6b7-3f12b7316988.png)

* line 5 had changed – `https://github.com/openstack-on-freebsd` is correct – during the hour or so that I spent editing. 

Since then, I got myself quite confused, I probably need to forget about it!
